### PR TITLE
[MPACT][CMAKE] Fix build errors.

### PR DESCRIPTION
**Pull request by PeimingLiu**

### CODE REVIEW

File: python/CMakeLists.txt [hunk 1]
```diff
@@ -21,7 +21,17 @@ declare_mlir_python_sources(MPACTPythonSources.PublicAPI
...
   SOURCES
     mpactbackend.py
-  )
+)
+
+declare_mlir_python_sources(MPACTPythonSources.SampleModels
+  ROOT_DIR "${MPACT_PYTHON_ROOT_DIR}"
+  ADD_TO_PARENT MPACTPythonSources
+  SOURCES_GLOB
+    models/gcn.py
```
Comment (yinying-lisa-li) [inline python/CMakeLists.txt:30]: Will models/*.py cause any issue?

Comment (PeimingLiu) [reply python/CMakeLists.txt:30]: most likely not, but I don't want to do another clean rebuild today ;-(

Comment (PeimingLiu) [reply python/CMakeLists.txt]: NVM, updated

File: CMakeLists.txt [hunk 2]
```diff
@@ -39,6 +39,9 @@ include_directories(${LLVM_INCLUDE_DIRS})
 include_directories(${MLIR_INCLUDE_DIRS})
 include_directories(${CMAKE_CURRENT_SOURCE_DIR}/include)
 include_directories(${CMAKE_CURRENT_BINARY_DIR}/include)
+# Needed to build TorchMLIRExtensions.
+# TODO: find a cleaner way
```
Comment (aartbik) [inline CMakeLists.txt:43]: just remove this comment completely, seems clean enough to me

**Review by aartbik — APPROVED** [review]

**Review by yinying-lisa-li — APPROVED** [review]

**Review by yinying-lisa-li — APPROVED** [review]

**Pull request merged**